cedar-policy-validator/src/schema.rs

```diff
@@ -3232,6 +3448,15 @@ mod test_rfc70 {
         }
     }
 
+    #[track_caller]
+    fn assert_invalid_json_schema(json: serde_json::Value) {
+        match ValidatorSchema::from_json_value(json.clone(), Extensions::all_available()) {
+            Ok(_) => panic!("{json} should be an invalid schema"),
+            Err(SchemaError::JsonDeserialization(_)) => {}
+            Err(e) => panic!("unexpected error: {:?}", miette::Report::new(e)),
```

Comment:
I think we'll want to add `[dev-dependencies] miette = { version = "7.1.0", features = ["fancy"] }` for this to pretty print the error when testing the validator. (It might work as is if it also happens to build the `fancy` feature for the CLI, but that's not guaranteed depending on how you run things)

Reply:
Done.